ddtrace/contrib/boto/patch.py

```diff
@@ -80,10 +80,9 @@ def patched_query_request(original_func, instance, args, kwargs):
 
         # Adding the args in AWS_QUERY_TRACED_ARGS if exist to the span
         if not aws.is_blacklist(endpoint_name):
-            for arg in aws.unpacking_args(
-                args, AWS_QUERY_ARGS_NAME, AWS_QUERY_TRACED_ARGS
-            ):
-                span.set_tag(arg[0], arg[1])
+            operation_args = aws.unpacking_args(args, AWS_QUERY_ARGS_NAME, AWS_QUERY_TRACED_ARGS)
+            for (key, value) in aws.flatten_args(operation_args):
+                span.set_tag('{}.{}'.format(span.resource, key), value)
```

Comment:
We don't want to do truncating here too?

Reply:
I wanted to keep not add any changes to how the value in this round of changes and instead focus on "flattening" the args. Only with `aiobotocore` was the `value` truncated previously, so I maintained the behavior.